pkg/ddevapp/webserver_config_assets/nginx-site-typo3.conf

```diff
@@ -23,6 +29,29 @@ server {
     error_log /dev/stdout info;
     access_log /var/log/nginx/access.log;
 
+    # Security: Content-Security-Policy
```

Comment:
`Content-Security-Policy` rules are in line with the autogenerated settings in `.htaccess` for Apache web servers.